Description: Fix a syntax error in Presentation.qml which prevents the slideshow from displaying
Author: Simon Quigley <tsimonq2@lubuntu.me>
Origin: vendor
Last-Update: 2024-10-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/qml/calamares-qt6/slideshow/Presentation.qml
+++ b/src/qml/calamares-qt6/slideshow/Presentation.qml
@@ -196,12 +196,7 @@ Item {
                 Text {
                     id: notesText
 
-                    property real padding: 16;
-
-                    x: padding
-                    y: padding
-                    width: parent.width - 2 * padding
-
+                    anchors.margins: 16
 
                     font.pixelSize: 16
                     wrapMode: Text.WordWrap
